core/src/components/datetime/datetime.scss

```diff
@@ -204,6 +204,45 @@ ion-picker-column-internal {
 // Calendar / Header / Action Buttons
 // -----------------------------------
 
+.calendar-month-year-toggle {
+  @include font-smoothing();
+  @include margin(0);
+  @include padding(0px, 0px, 0px, 20px);
+  @include text-inherit();
```

Comment:
Hover states are missing too. However, they are missing on iOS mode in `main` today